content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -23,6 +23,89 @@ Spring Boot starter, see
 
 ## Configuration
 
+### Bill of Materials
+
+Import the `opentelemetry-bom` and `opentelemetry-instrumentation-bom-alpha`
+Bill of Materials ([BOM](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms)) before using the OpenTelemetry starter.
+
+The `opentelemetry-bom` BOM will allow you to align the OpenTelemetry versions
+of the OpenTelemetry dependencies and transitive dependencies.
+
+With the `opentelemetry-instrumentation-bom-alpha` BOM, you won't have to
+declare the dependency versions for additional instrumentations.
```

Comment:
tried to simplify a bit and make it more beginner friendly, not sure if I succeeded...

```suggestion
### Dependencies

Importing the `opentelemetry-bom` and `opentelemetry-instrumentation-bom-alpha` BOMs will ensure that the versions of your OpenTelemetry dependencies (and transitive dependencies) are aligned.
```

Reply:
I prefer this version. Thanks!

Reply:
@svrnm What do you think? Would it be good enough for merging?

Reply:
What's missing for me: the term BOM needs to be written out when used for the first time Bill Of Materials (BOM), and then some light reason what it is (in general terms) and why someone wants to have that, e.g.

```
A Bill of Materials (BOM) is a summary of the components used within a software artifact, which helps with compliance and security. By importing the ...
```

Reply:
@svrnm Could you please check the new version?

Reply:
lgtm now. @trask any more comments?